### ISSUE TITLE
Error in GUI with v6.6.5, js controller 5.0.11

### ISSUE THREAD

**Issue by beandi** [ioBroker/ioBroker.admin#2027]
**Describe the bug**  
"Error in GUI"-Message, only on the hosts-tab (multihost), after approx. 5 seconds

**To Reproduce**  
Just clicking hosts

**Screenshots & Logfiles**  
`Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7BisDocker%2C%20isOfficial%2C%20officialVersion%7D for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    React 11
        Ka
        d
        h
        e
        wo
        Ei
        ks
        ys
        vs
        us
        as
    S scheduler.production.min.js:13
    T scheduler.production.min.js:14
    36813 scheduler.production.min.js:14
    Webpack 18
        t
        45296
        t
        34463
        t
        54164
        t
        I
        e
        t
        1250
        t
        16677
        t
        promise callback*77569
        t
        <anonymous>
        <anonymous>
react-dom.production.min.js:189:29
    React 9
        fo
        callback
        Ua
        wi
        ki
        bi
        Ss
        Ss
        as
    S scheduler.production.min.js:13
    T scheduler.production.min.js:14
    (Async: EventHandlerNonNull)
    36813 scheduler.production.min.js:14
    Webpack 18
        t
        45296
        t
        34463
        t
        54164
        t
        I
        e
        t
        1250
        t
        16677
        t
        (Async: promise callback)
    77569
        t
        <anonym>
        <anonym>
`

**Versions:**  
 - Adapter version: 6.6.5
 - JS-Controller version: 5.0.11

**Comment by winnyschuster** [comment]
i can confirm all of above. In addition, error was introduced in conjunction with controller 5.0.11, versions before do not show this error. Also, i am using Docker(official 8.0.1 but also 8.1.0-beta.4). When setting chrome debugger to stop on known exceptions, i can see that check for docker was successfull.

Below is what is shown **before error message** , error is same like above so not included. Last line `translate: Dockerinformation` is new since 5.0.11

```bootstrap.jsx:20 iobroker.admin@6.6.5
15:14:50.211 instrument.ts:132 [2023-08-02T13:14:50.210Z] Try to connect
15:14:50.405 instrument.ts:132 Stored version: , new version: 6.6.5
15:14:50.482 instrument.ts:132 Connected: true
15:14:53.619 fetch.ts:45 Laden von Abrufen beendet: POST „https://sentry.iobroker.net/api/58/envelope/?sentry_key=43643152dab3481db69950ba866ee9d6&sentry_version=7&sentry_client=sentry.javascript.browser%2F7.61.0“.
(anonym) @ fetch.ts:45
(anonym) @ base.ts:76
add @ promisebuffer.ts:49
i @ base.ts:92
value @ baseclient.ts:700
value @ baseclient.ts:346
value @ baseclient.ts:218
value @ hub.ts:472
value @ hub.ts:404
Bn @ sdk.ts:255
(anonym) @ sdk.ts:284
Yn @ sdk.ts:131
16677 @ bootstrap.jsx:70
t @ bootstrap:19
Promise.then (async)
77569 @ index.jsx:1
t @ bootstrap:19
(anonym) @ startup:4
(anonym) @ main.a542f0b9.js:1
15:14:55.447 instrument.ts:132 Translate: time
15:14:55.447 instrument.ts:132 Translate: timeOffset
15:14:55.447 instrument.ts:132 Translate: dockerInformation
15:16:51.028 instrument.ts:132 Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7BisDocker%2C%20isOfficial%2C%20officialVersion%7D for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Ka (react-dom.production.min.js:152:60)
    at d (react-dom.production.min.js:156:186)
    at h (react-dom.production.min.js:158:331)
    at e (react-dom.production.min.js:162:334)
    at wo (react-dom.production.min.js:192:308)
    at Ei (react-dom.production.min.js:294:9)
    at ks (react-dom.production.min.js:280:389)
    at ys (react-dom.production.min.js:280:320)
    at vs (react-dom.production.min.js:280:180)
    at us (react-dom.production.min.js:271:88)
    at as (react-dom.production.min.js:268:429)
    at S (scheduler.production.min.js:13:203)
    at MessagePort.T (scheduler.production.min.js:14:128)bootstrap.jsx:20 iobroker.admin@6.6.5
15:14:50.211 instrument.ts:132 [2023-08-02T13:14:50.210Z] Try to connect
15:14:50.405 instrument.ts:132 Stored version: , new version: 6.6.5
15:14:50.482 instrument.ts:132 Connected: true
15:14:53.619 fetch.ts:45 Laden von Abrufen beendet: POST „https://sentry.iobroker.net/api/58/envelope/?sentry_key=43643152dab3481db69950ba866ee9d6&sentry_version=7&sentry_client=sentry.javascript.browser%2F7.61.0“.
(anonym) @ fetch.ts:45
(anonym) @ base.ts:76
add @ promisebuffer.ts:49
i @ base.ts:92
value @ baseclient.ts:700
value @ baseclient.ts:346
value @ baseclient.ts:218
value @ hub.ts:472
value @ hub.ts:404
Bn @ sdk.ts:255
(anonym) @ sdk.ts:284
Yn @ sdk.ts:131
16677 @ bootstrap.jsx:70
t @ bootstrap:19
Promise.then (async)
77569 @ index.jsx:1
t @ bootstrap:19
(anonym) @ startup:4
(anonym) @ main.a542f0b9.js:1
15:14:55.447 instrument.ts:132 Translate: time
15:14:55.447 instrument.ts:132 Translate: timeOffset
15:14:55.447 instrument.ts:132 Translate: dockerInformation
```
Below find known exception which comes before error message
![DockerCheck](https://github.com/ioBroker/ioBroker.admin/assets/7774086/1ee780a7-291e-4fbf-8e6c-4381254d02b4)

**Comment by foxriver76** [comment]
Fixed on 6.7.0

**Comment by winnyschuster** [comment]
i can confirm it's fixed. Like the info about image version!
